sklearn/cluster/k_means_.py

```diff
@@ -792,6 +792,21 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
     inertia_ : float
         Sum of distances of samples to their closest cluster center.
 
+    Examples
+    --------
+
+    >>> from sklearn.cluster import KMeans
+    >>> import numpy as np
+    >>> X = np.array([[1, 2], [0, -4], [9, 1], [0, 0]])
```

Comment:
Also could you add more number of samples? Essentially you should be showing that a data with some similar points get clustered to the same label.

Like 
```
[[1, 2], [1, 3], [0, 3],
 [4, 3], [4, 4], [5, 1]]
```

gets clustered to 

`labels_ = [0, 0, 0, 1, 1, 1]`